src/components/Input.tsx

```diff
@@ -21,19 +22,22 @@ const Input = ({
       >
         {label}
       </label>
-      <input
-        type={type}
+      <Field
         name={name}
-        id={name}
```

Comment:
why removed id?

Reply:
for now, we don't need this. If we need this in future then will add it.